README.md

```diff
@@ -1,54 +1,316 @@
-# Welcome to WOVN.php
+# WOVN.php Instructions
 
-## I. Basic install
-### 1. Clone the repository in your application root folder
-### 2. Edit the configuration file
-These are the three main settings for your project
- - project_token: your WOVNio's project token
- - url_pattern_name: query, path or subdomain (default is query)
- - default_lang: the code of the default language of your website, ex: ja (for Japanese)
+## Table of Contents
+1. [Requirements](#1-requirements)
+2. [Getting Started](#2-getting-started)
+3. [Configuration](#3-configuration)
+4. [Bug Report](#4-bug-report)
+5. [Troubleshooting](#5-troubleshooting)
 
-Please copy the sample file from the library `wovn.ini.sample`, rename it to `wovn.ini` and paste it in your framework's root directory (same level as WOVN.php directory)
+## 1. Requirements
+WOVN.php requires PHP 5.3 or higher. WOVN.php has no third-party dependencies.
+Depending on your configuration, you might have to install and/or activate the
+PHP module `mod_rewrite` (see [Section 2.3.2.](#232-for-static-websites) and
+[Section 3.2.](#32-optional-parameters)).
 
-After pasting, your directory structure should look like the following:
+WOVN.php has been tested with Apache 2 and Nginx. We provide installation
+instructions for both. If you use other technologies, we encourage you to
+[contact us](mailto:support@wovn.io) for support.
+
+## 2. Getting Started
+### 2.1. Download WOVN.php
+To install WOVN.php, you have to manually download WOVN.php from our Github
+repository. The root directory of WOVN.php must be place  at the root of your
+website's directory. In the rest of this document, we consider the root
+directory of your website to be `/website/root/directory`.
+```
+$ cd /website/root/directory
+$ wget https://github.com/WOVNio/WOVN.php/archive/master.zip -O WOVN.php.zip
+$ unzip WOVN.php.zip; mv WOVN.php-master WOVN.php
+```
+
+**Note on updates:** When you need to update WOVN.php, you can simply replace
+all the content of `WOVN.php` directory by the content of the new version.
```

Comment:
"replace with" would be correct, I think
`directory by the content of` -> `directory with the content of`

Reply:
done, thanks